src/moon.c

```diff
@@ -0,0 +1,225 @@
+#include <predict/predict.h>
+#include "unsorted.h"
+#include <stdlib.h>
+#include <string.h>
+#include "defs.h"
+#include "sun.h"
+
+/**
+ * This function reduces angles greater than two pi by subtracting two pi
+ * from the angle
+ *
+ * \copyright GPLv2+
+ **/
```

Comment:
In case you didn't know, it's not necessary to use **/ to end the comment. */ is enough.